### DISCUSSION TITLE
[BUG] 网页发信失败，400 Please enable SEND_MAIL to use verfiedAddressList 和 400 Failed to send mail Please enable resend or verified address list

### DISCUSSION THREAD

**Discussion by 1834423612** [dreamhunter2333/cloudflare_temp_email#423] [General]
## 复现步骤

【报错页面】
![image](https://github.com/user-attachments/assets/5ba8fa8c-e510-469c-89d1-92fdd7349ef4)

![image](https://github.com/user-attachments/assets/835c135a-1420-47ac-8319-77ea4ea452ce)

----  

<details>
<summary>【cloudflare 配置页面】</summary>
----  

![image](https://github.com/user-attachments/assets/80795a49-5bdf-4373-ad72-ca520b5eb426)

![image](https://github.com/user-attachments/assets/53f5b708-1def-41fd-b8ee-292f1d40b812)

![image](https://github.com/user-attachments/assets/27f9dc4e-b533-43f2-b3e0-828d27aa39f2)

![image](https://github.com/user-attachments/assets/ed95ec0c-5d08-4854-a040-7084b7011aaf)

![image](https://github.com/user-attachments/assets/7068558f-d570-4eb7-8a0a-259a46643c90)

</details>

----  

<details>
<summary>【Admin后台账号设置页面】</summary>
----  

![image](https://github.com/user-attachments/assets/20dea312-6ae2-40cd-a118-d10a68e40a86)

![image](https://github.com/user-attachments/assets/13176079-f6b0-418e-a5ce-e78f8b5a2b96)

</details>

----  

所有根域名一样，二级域名和三级mail域名均无法使用。三级域名mail可以收件，但无法发件。
请问这是bug还是需要额外配置什么吗？如果需要配置，要如何去配置？
谢谢大佬！！！

## 预期行为

预期会正常像示例网站一样通过网页进行发送与回复或接收邮件

## 部署方式

- [ ] cli 部署
- [x]  用户界面部署

## 浏览器环境

Chrome 126.0.0.0
Windows 11 专业版 23H2 22631.3880 x64位

**Comment by dreamhunter2333** [comment]
感谢反馈

和提示的一样，给 cf 中验证的邮箱发件时需要在 admin 后台配置 cf 的验证邮箱，并且通过 cli 部署绑定 send mail

给通用的其他邮箱发邮件请配置 resend

不清楚 resend 是否支持非根域名

**Comment by 1834423612** [comment]
谢谢！！！
我试一下

**Comment by Larry-znhy** [comment]
> 不清楚 resend 是否支持非根域名

亲测非根域名也支持

**Comment by dreamhunter2333** [comment]
> > 不清楚 resend 是否支持非根域名
> 
> 
> 
> 亲测非根域名也支持
> 
> 

赞
